doc/tor.md

```diff
@@ -211,3 +209,102 @@ for normal IPv4/IPv6 communication, use:
   Otherwise it is trivial to link them, which may reduce privacy. Onion
   services created automatically (as in section 2) always have only one port
   open.
+
+- In some regions of the world usage of the Tor network is monitored and/or blocked.
+  If you are in an environment that does not permit direct Tor connections or the use
+  of Tor bridges, then considering the trade-offs, it may not be safe to use Tor.
+  Another option for Bitcoin network privacy is to use I2P (option `i2p`).
```

Comment:
What is I2P (option `i2p`)?

Reply:
I2P is the Invisible Internet Project, a second privacy network that was added to Bitcoin Core for the upcoming release￼￼.

I've written some docs for an initial doc/i2p.md file her , just need to overcome my desire to avoid bikeshedding to propose it 😛

Reply:
@jonatack You know more about this PR, Tor and related things. I am closing it. I am not Blue but you can always use things in a new PR if it helps Core. I won't ever mislead people about Core.